Dockerfile

```diff
@@ -9,6 +9,12 @@ COPY package.json package-lock.json ./
 # Install Node Packages
 RUN npm install
 
+# Copying self-signed certs
+COPY certs/ certs/
+
+# Generate certs
+RUN cd certs && ./certgen.sh && cd ..
```

Comment:
The final `cd ..` is unnecessary. Every `RUN` runs in a subshell and as such the current directory is not preserved unless `WORKDIR` is used.

Reply:
TIL

Reply:
(it all makes sense if you consider that the current directory is an environment variable in a certain way!)